src/StatusQ/Components/StatusClickableTag.qml

```diff
@@ -0,0 +1,139 @@
+import QtQuick 2.13
+import QtQuick.Controls 2.12
+import QtQuick.Layouts 1.12
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+
+/*!
+   \qmltype StatusAccountSelectorTag
+   \inherits Rectangle
+   \inqmlmodule StatusQ.Components
+   \since StatusQ.Components 0.1
+   \brief The StatusAccountSelectorTag provides a template for creating a clickable tag item.
+
+   Example of how to use it:
+
+   \qml
+        StatusClickableTag {
+            title: "some name"
+            icon.name: "some icon name"
+            isSelected: false
+            onClicked: {
+                // do something
+            }
+        }
```

Comment:
Maybe I'm wrong, but to me this looks like just another variation of `StatusButton`.
Or at least, the one question that comes up is: Why is this considered a "tag" and not a "button" ?

Reply:
hmm good point, I think it could be a button just different look and feel